src/common/components/clusterConfiguration/DiscoveryImageConfigForm.tsx

```diff
@@ -126,6 +129,7 @@ export const DiscoveryImageConfigForm: React.FC<DiscoveryImageConfigFormProps> =
                     {!hideDiscoveryImageType && <DiscoveryImageTypeControlGroup />}
                     <UploadSSH />
                     <ProxyFields />
+                    <CertificateFields />
```

Comment:
We'll need to confirm with CIM if we want to include the Certificates field also for them or not.

Reply:
Then the two issues need to be accordingly linked and we must hold this PR until the other one is merged.